armi/bookkeeping/report/reportingUtils.py

```diff
@@ -139,13 +139,21 @@ def _listInputFiles(cs):
 
         # bonus: grab the files stored in the crossSectionControl section
         for fluxSection, fluxData in cs["crossSectionControl"].items():
```

Comment:
we don't *have* to fix it in this PR, but the pre-existing variable names of `fluxSection` and `fluxData` here are confusing to me. I would propose we replace them with:

`fluxSection` -> `xsID`
`fluxData` -> `xsSettings`  (open to suggestions for this one)

Reply:
Agreed. Not sure if now is the time. But agreed.

Reply:
oh you alrady did it, even better!